packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -419,10 +419,13 @@ export class ProfilesCache {
      * not initialized it we mock a default value.
      */
     private getZoweDir(): string {
-        imperative.ImperativeConfig.instance.loadedConfig = {
-            defaultHome: path.join(os.homedir(), ".zowe"),
-            envVariablePrefix: "ZOWE",
-        };
+        if (imperative.ImperativeConfig.instance.loadedConfig == null) {
```

Comment:
If there are concerns about setting loadedConfig only once (e.g. the `os.homedir()` may change from command to command ?) ...

We can replace 
```javascript
if (imperative.ImperativeConfig.instance.loadedConfig == null)
```
for
```javascript
if (imperative.ImperativeConfig.instance.loadedConfig?.defaultHome !== path.join(os.homedir(), ".zowe"))
```
---
A better approach might be to replace this private function with [`ProfileUtils.getZoweDir()`](https://github.com/zowe/zowe-cli/blob/next/packages/core/src/utils/ProfileUtils.ts#L20) and fix this same issue there. 😋

Reply:
Removed private function and fixed the issue in https://github.com/zowe/zowe-cli/pull/1344